bot/exts/fun/uwu.py

```diff
@@ -116,10 +61,57 @@ class Uwu(Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
+    def _word_replace(self, input_string: str) -> str:
+        """Replaces words that are keys in the word replacement hash to the values specified."""
+        for word in WORD_REPLACE_HASH:
+            input_string = input_string.replace(word, WORD_REPLACE_HASH[word])
```

Comment:
Might as well use `dict.items()` like this:
```suggestion
        for word, replacement in WORD_REPLACE_HASH.items():
            input_string = input_string.replace(word, replacement)
```